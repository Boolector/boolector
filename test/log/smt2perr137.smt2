(
assert
(
(
_
rotate_right
10
			       unexpected_symbol

; some more lines
