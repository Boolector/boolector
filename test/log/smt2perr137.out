log/smt2perr137.smt2:8:11: expected ')' to close '(_ rotate_right' at 'unexpected_symbol'
